docker/testnet/docker-compose.testnet.yml

```diff
@@ -0,0 +1,47 @@
+services:
+  postgres:
+    image: postgres:16.3-alpine
+    stop_grace_period: 5s
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
```

Comment:
we did remove the port exposure, should we also suggest to use some different password here?

Reply:
We could, but then users would need to change the user/config on the provided PG dump as well.

Reply:
If we are talking only about the password then it's fine: pgdump will include statements like `ALTER SCHEMA sbtc_signer OWNER TO postgres;`, but not the `postgres` user itself; this applies for "proper" pgdumps (which we should start to use instead of the disk image).
The would still need to update the db config. Maybe we can mention it on the readme (_if you want, you can change the db password and update here and here_).

Reply:
What about setting it in `.env`, and then document that they'd only have to change it there? It can be then reused in the compose file and config, example:
```sh
# .env
PG_PASSWORD=postgres # PostgreSQL Database Password, modify this to a password of your choice
```

```yml  
# docker-compose.yml
services:
  postgres:
    environment:
      POSTGRES_PASSWORD: ${PG_PASSWORD:-postgres}

  sbtc-signer:
    environment:
      PG_PASSWORD: ${PG_PASSWORD:-postgres}
    volumes:
      - ./config/signer-config.toml:/signer-config.toml.in
    entrypoint: 
      - /bin/bash
      - -c
      - |
        envsubst < /signer-config.toml.in > /signer-config.toml
        /usr/local/bin/signer -c /signer-config.toml --migrate-db -o json | tee /logs/sbtc_signer.logs
```

```toml
# signer-config.toml
[signer]
db_endpoint = "postgresql://postgres:$PG_PASSWORD@postgres:5432/signer"
```